hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/info/FakeNetworkInfo.java

```diff
@@ -107,7 +107,7 @@ public AccountID accountId() {
             }
 
             @Override
-            public long stake() {
+            public long weight() {
                 return stake;
```

Comment:
I'm curious about the method name change?    it makes it sound like consensus weight, but its the stake value?